specs/system_config.md

```diff
@@ -39,20 +39,49 @@ The L1 fee parameters, also known as Gas Price Oracle (GPO) parameters, are used
 data fee applied to an L2 transaction.  The specific parameters used depend on the upgrades that
 are active.
 
-#### `overhead`,`scalar` (`uint256,uint256`)
+Fee parameter updates are signaled to L2 through the `GAS_CONFIG` log-event of the `SystemConfig`.
 
-Prior to the Ecotone upgrade, `overhead` and `scalar` are consulted and passed to the L2 via L1
-attribute info.
+#### Pre-Ecotone `scalar`, `overhead` (`uint256,uint256`)
 
-#### `l1BasefeeScalar`,`l1BlobBasefeeScalar` (`uint32,uint32`)
+The `overhead` and `scalar` are consulted and passed to the L2 via L1 attribute info.
 
-After the Ecotone upgrade, `l1BasefeeScalar` and `l1BlobBasefeeScalar` are passed to the L2
-instead.
+The values are interpreted as big-endian `uint256`.
 
-The only exception is for chains that have genesis prior to Ecotone and go through the Ecotone
-transition. For these chains, the very first Ecotone block will pass the older
-parameters. Thereafter and up until a type `4` log event is processed, `l1BasefeeScalar` passed to
-the L2 *must* be set to the value of `scalar`, or MaxUint32 if `scalar` is outside 32-bit range.
+#### Ecotone `scalar`, `overhead` (`uint256,uint256`) change
+
+After Ecotone activation, the `scalar` attribute encodes additional scalar information, in a versioned encoding scheme.
+
+After Ecotone activation, the `overhead` value is ignored: it does not affect the L2 state-transition output.
+
+The `scalar` is encoded as big-endian `uint256`. This `bytes32` is decoded as following:
```

Comment:
I found this a little confusing: you say it's encoded as big-endian `uint256` and then immediately write "This `bytes32` ..."     Maybe say more explicitly it's a uint256 that is interpreted as `bytes32` after upgrade?

Reply:
Sure